packages/atst/src/__snapshots__/logger.spec.ts.snap

```diff
@@ -0,0 +1,11 @@
+// Vitest Snapshot v1
+
+exports[`logger > \${level}() > logs message "error" 1`] = `"[31merror[39m"`;
```

Comment:
Will those logs change? Just to understand if we are committing snapshot that are likely to bark at us

Reply:
Not these ones.   These are just testing the logger.   Also, I could  just nuke these if I wanted.   These tests and this logger are 100% copy paste from wagmi